### PR TITLE
Add r-iccbeta

### PR DESCRIPTION
**Pull request by coatless**
cc @conda-forge/help-r

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-iccbeta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-rcpp
+    - r-rcpparmadillo >=0.9.200
+    - r-lme4
```
Comment (dbast) [inline recipes/r-iccbeta/meta.yaml:39]: Requires libblas:
```suggestion
    - r-lme4
    - libblas
```

**Comment by dbast** [comment]
@steveculpepper Do you agree being co-maintainer?

**Comment by coatless** [comment]
@dbast he's fine with it. Though, I'll likely be handling the updates if needed.

**Comment by steveculpepper** [comment]
Yes. Thank you.

Sent from my iPhone

On Jan 4, 2021, at 12:41, Daniel Bast <notifications@github.com> wrote:

﻿

@steveculpepper<https://urldefense.com/v3/__https://github.com/steveculpepper__;!!DZ3fjg!rCHbWR5sAaS6921-xpQ2vmrcVV7DdTOtbDiNUi4Oi_wmNCZlof-f5kxG5-lfPFjQNLI$> Do you agree being co-maintainer?

**Pull request merged**